node/src/effect.rs

```diff
@@ -1756,6 +1714,6 @@ impl<REv> EffectBuilder<REv> {
 #[macro_export]
 macro_rules! fatal {
     ($effect_builder:expr, $($arg:tt)*) => {
-        $effect_builder.fatal(file!(), line!(), format_args!($($arg)*).to_string())
+        $effect_builder.fatal(file!(), line!(), format!($($arg)*).to_string())
```

Comment:
This should be redundant now?
```suggestion
        $effect_builder.fatal(file!(), line!(), format!($($arg)*))
```

Reply:
Fixed in a9f3fcfce4f7b72bd1577e401c322cdf6c057e03